docs/getting_started.md

```diff
@@ -58,12 +96,15 @@ In a python shell:
 See the [wiki](https://eth.wiki/json-rpc/API#json-rpc-methods) for other standard methods that are implemented. You can use the [web3.py](https://web3py.readthedocs.io/en/stable/web3.eth.html#module-web3.eth) API to access these. Note that currently most of them proxy to Infura rather than requesting the data from the Portal Network.
```

Comment:
Re: the earlier point about trin not actually proxying all of the ethereum jsonrpc endpoints to infura.

Reply:
Yeah, good callout. Are you recommending that we replace this note with the list of endpoints that we currently support? I guess that would just be `web3_clientVersion` and `eth_blockNumber`?